### PR TITLE
CSHARP-4171: Support for Deploying MongoDB behind a L4 Load Balancer 5.0 Backports.

### PR DESCRIPTION
**Pull request by DmitryLukyanov**
EG: https://evergreen.mongodb.com/version/6286ac582fbabe0be724d3b0

### CODE REVIEW

File: evergreen/evergreen.yml [hunk 1]
```diff
@@ -1600,14 +1600,14 @@ buildvariants:
     - name: plain-auth-tests
 
 - matrix_name: load-balancer-tests
-  matrix_spec: { version: ["latest"], auth: "noauth", ssl: "nossl", topology: "sharded-cluster", os: "ubuntu-1804" }
+  matrix_spec: { version: ["5.0", "6.0", "latest"], auth: "noauth", ssl: "nossl", topology: "sharded-cluster", os: "ubuntu-1804" }
```
Comment (JamesKovacs) [inline evergreen/evergreen.yml:1603]: We should include the `rapid` releases as well.

File: evergreen/evergreen.yml [hunk 2]
```diff
@@ -1600,14 +1600,14 @@ buildvariants:
...
   display_name: "Load Balancer ${version} ${auth} ${ssl} ${os}"
   tasks:
     - name: "test-load-balancer-netstandard20"
     - name: "test-load-balancer-netstandard21"
 
 - matrix_name: load-balancer-tests-secure
-  matrix_spec: { version: ["latest"], auth: "auth", ssl: "ssl", topology: "sharded-cluster", os: "ubuntu-1804" }
+  matrix_spec: { version: ["5.0", "6.0", "latest"], auth: "auth", ssl: "ssl", topology: "sharded-cluster", os: "ubuntu-1804" }
```
Comment (JamesKovacs) [inline evergreen/evergreen.yml:1610]: Same as above.

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Minor change requested.

File: evergreen/evergreen.yml (hunk 1 above)
Comment (DmitryLukyanov) [reply evergreen/evergreen.yml]: done

File: evergreen/evergreen.yml (hunk 2 above)
Comment (DmitryLukyanov) [reply evergreen/evergreen.yml]: done

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**